### PR TITLE
Add debug in secrets handler to investigate staging

### PR DESCRIPTION
**Pull request by redhug1**

### CODE REVIEW

File: handler/secret/secret.go [hunk 1]
```diff
@@ -68,10 +69,12 @@ func (s *Secret) Handler(ctx context.Context, msg *engine.Message) error {
 
 			d, err := s.decryptMessage(b)
 			if err != nil {
+				log.Info(ctx, "Secret-Handler, s.decryptMessage(b)", log.Data{"err": err})
 				return err
 			}
 			log.Info(ctx, "writing secret", log.Data{"artifact": artifact})
 			if err := s.write(pathFor(artifact), d); err != nil {
+				log.Info(ctx, "Secret-Handler, s.write(pathFor", log.Data{"err": err})
```
Comment (gedge) [inline handler/secret/secret.go:77]:
```suggestion
				log.Info(ctx, "Secret-Handler, s.write(pathFor)", log.Data{"err": err})
```

File: handler/secret/secret.go [hunk 2]
```diff
@@ -60,6 +60,7 @@ func (s *Secret) Handler(ctx context.Context, msg *engine.Message) error {
 		default:
 			b, _, err := s.s3Client.Get(artifact)
 			if err != nil {
+				log.Info(ctx, "Secret-Handler, s.s3Client.Get(artifact) error", log.Data{"err": err})
```
Comment (gedge) [inline handler/secret/secret.go:63]:
```suggestion
				log.Error(ctx, "Secret-Handler, s.s3Client.Get(artifact) error", err)
```

File: handler/secret/secret.go (hunk 1 above)
Comment (redhug1) [reply handler/secret/secret.go:77]: adjusted as per slack message.

File: handler/secret/secret.go (hunk 2 above)
Comment (redhug1) [reply handler/secret/secret.go:63]: adjusted as per slack message.

**Review by gedge — APPROVED** [review]

**Pull request merged**